test/RifleVampireTests.cpp

```diff
@@ -243,8 +279,191 @@ void RifleVampireTests::OneRifleNVampiresBenchmark(int nVampires, int nIOThreads
    delete rifle;
 }
 
+
+  template <typename Sender>
+   size_t Push(Sender q, int dataSize, int stop, std::atomic<bool>& stopRunning) {
```

Comment:
I'll remove all the changes to this file since I added performance tests elsewhere.

Reply:
Hmm. I'm want to leave these in for now since it's a general cleanup and for rifle vampire comparisons in case there would be discussions about it later.  

Since the file will be deleted hopefully fairly soon I would like to leave as is without spending more time on this. 

... 
@craig-cogdill  if you feel strongly about this I'd rather just remove my changes in this file. I'll let you make the call.